src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -1034,6 +1043,10 @@ class AddItemsPage extends Component {
     // rowIndex is present
     const isEdited = rowIndex !== undefined;
     const itemsWithStatuses = values.lineItems.map((item) => {
+      if (editAll) {
+        return { ...item, rowSaveStatus: RowSaveStatus.PENDING };
+      }
+
```

Comment:
I can see that the line below has an additional condition when `RowSaveStatus.PENDING` is applied. Have you checked what will happen when editing the same row multiple times?

Reply:
in that regard everything should work properly.
Addityionaly I have added the `editAll` condition to the following statement.
```jsx
if (isEdited || editAll) {
  this.debouncedSave();
  return;
}
```
this should ensure a safe items saveing step if there are pending save processes from previous actions.